src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -139,3 +139,21 @@ func AddAdminAPITLSFlags(
 
 	return command
 }
+
+func AddBrokersFlag(
+	command *cobra.Command,
+        brokers *[]string,
+) *cobra.Command {
+        command.PersistentFlags().StringSliceVar(
+                brokers,
+                "brokers",
+                []string{},
+                "Comma-separated list of broker ip:port pairs (e.g."+
```

Comment:
nit: brokers

Reply:
Fixed!

Reply:
We're planning to change some of this text soon, but at the moment this should still be "broker", which is a bit more obvious if we drop the ip:port descriptive text: broker pairs vs brokers pairs.

Reply:
Fixed!